package.json

```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "fastify-plugin": "^4.0.0",
-    "vary": "^1.1.2"
+    "mnemonist": "^0.39.2"
```

Comment:
This seems like a really heavy dependency. It also isn't at a stable release number.